client/src/components/J40Map.module.scss

```diff
@@ -30,10 +30,16 @@ $sidebar-color: #ffffff;
   line-height: 40px;
   position: absolute;
   left: 20px;
-  top: 300px;
+  top: 500px;
   z-index: 10;
 }
 
+@media screen and (max-width: 400px){
```

Comment:
Good catch!

Reply:
A side note, we may need an @media query for "phablet" as well - at 500px I was seeing it jump as well. I realize this bug was actually introduced with my latest fix, however, in using absolute positioning. We can / should revisit this later

Reply:
agreed, there's a part of me that wants to fix most of fixed px values